### PR TITLE
Error with stop words in exact match searches

### PR DESCRIPTION
**Pull request by guyroyse**
- Fixed error message with stop words in exact match searches.
- Added ability to override or disable stop words.

### CODE REVIEW

File: docs/classes/Schema.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ let schema = new Schema(Foo, {
   aBoolean: { type: 'boolean' },
   anArray: { type: 'array' }
 }, {
-  dataStructure: 'JSON'
+  dataStructure: 'JSON
```
Comment (justincastilla) [inline docs/classes/Schema.md:16]: whoops!

Comment (guyroyse) [reply docs/classes/Schema.md:16]: Odd. Simon just fixed that.

Comment (guyroyse) [reply docs/classes/Schema.md:16]: Thought I merged it.

Comment (guyroyse) [reply docs/classes/Schema.md:16]: Ah. Was fixed in the docs, which are generated, not the source.

**Review by justincastilla — APPROVED** [review]

**Pull request merged**